e-shop-manager/src/main/java/it/unipd/mtss/model/EItem.java

```diff
@@ -5,5 +5,25 @@
 package it.unipd.mtss.model;
 
 public class EItem {
-    
+    private EItemType itemType;
+    private String name;
+    private double price;
+
+    EItem(EItemType itemType, String name, double price){
```

Comment:
```suggestion
    public EItem(EItemType itemType, String name, double price){
```
sennò negli altri package non lo vede e da errore